### DISCUSSION TITLE
Resource routes should open in a new window

### DISCUSSION THREAD

**Discussion by houmark** [Code-Forge-Net/Remix-Dev-Tools#50] [Ideas]
On the routes tab, if you have a resource route and click the button to open the route, it will open in the same window, but data from the resource route is not shown as it requires a full browser reload to work.

P.S. Thanks for this, it's pretty cool :)

**Comment by AlemTuzlak** [comment]
I'll see if theres a way to know if the route is a resource or not, will look into this further!

**Comment by houmark** [comment]
Not sure how you are parsing for information, and you are probably well aware of this, but in case you're not, the basic definition of a resource route is one that does not export a default component, so it should be somewhat simple to detect I think.

**Comment by AlemTuzlak** [comment]
@houmark I'm fully aware, the issue is that the remix context doesn't expose this information until the route itself is mounted (which it's not on the routes page) so I would probably have to include remix forge into this to check beforehand or do some hacking so it's not really as simple and naive as it looks. But I have this in mind and will try to find a way to do it
